charts/timescaledb-single/templates/statefulset-timescaledb.yaml

```diff
@@ -34,6 +34,7 @@ spec:
 {{- end }}
     spec:
       serviceAccountName: {{ template "timescaledb.serviceAccountName" . }}
+      priorityClassName: {{ .Values.priorityClassName }}
```

Comment:
This should be conditionally set and present only if `.Values.priorityClassName` is not empty